app/i18n/webapp-cs.json

```diff
@@ -16,8 +16,41 @@
   "createAccount.passwordPlaceholder": "Heslo (alespoň 8 znaků)",
   "createAccount.terms": "",
   "createAccount.nextButton": "Další",
+  "invite.nextButton": "Další",
   "verify.headline": "",
   "verify.subhead": "",
   "verify.resendCode": "",
-  "verify.changeEmail": "Změnit email"
+  "verify.changeEmail": "Změnit email",
+  "invite.headline": "",
+  "invite.subhead": "",
+  "invite.emailPlaceholder": "",
+  "invite.skipForNow": "Prozatím přeskočit",
+  "BackendError.LABEL.ACCESS_DENIED": "Zkontrolujte své údaje a zkuste znovu.",
+  "BackendError.LABEL.INVALID_CREDENTIALS": "Zkontrolujte své údaje a zkuste znovu.",
+  "BackendError.LABEL.MISSING_AUTH": "Zkontrolujte své údaje a zkuste znovu.",
+  "BackendError.LABEL.BLACKLISTED_EMAIL": "Tato e-mailová adresa není povolena.",
+  "BackendError.LABEL.BLACKLISTED_PHONE": "",
+  "BackendError.LABEL.INVALID_CODE": "Tento kód je neplatný.",
+  "BackendError.LABEL.INVALID_EMAIL": "Tato emailová adresa je neplatná.",
+  "BackendError.LABEL.INVALID_PHONE": "",
+  "BackendError.LABEL.KEY_EXISTS": "",
+  "BackendError.LABEL.PENDING_ACTIVATION": "",
+  "BackendError.LABEL.PENDING_LOGIN": "",
+  "BackendError.LABEL.TOO_MANY_LOGINS": "Prosím, zkuste se přihlásit pozdějí.",
+  "BackendError.LABEL.BAD_REQUEST": "",
+  "BackendError.LABEL.INVALID_OPERATION": "",
+  "BackendError.LABEL.NOT_FOUND": "",
+  "BackendError.LABEL.OPERATION_DENIED": "",
+  "BackendError.LABEL.UNAUTHORIZED": "Něco se pokazilo. Prosíme, aktualizujte stránku a zkuste to znovu.",
+  "BackendError.LABEL.HANDLE_EXISTS": "",
+  "BackendError.LABEL.INVALID_HANDLE": "",
+  "BackendError.LABEL.INVALID_INVITATION_CODE": "",
+  "BackendError.LABEL.NO_OTHER_OWNER": "",
+  "BackendError.LABEL.NO_TEAM": "",
+  "BackendError.LABEL.NO_TEAM_MEMBER": "",
+  "BackendError.LABEL.TOO_MANY_MEMBERS": "",
+  "BackendError.LABEL.SUSPENDED": "Pro tento účet již není povoleno se přihlásit.",
+  "BackendError.LABEL.EMAIL_EXISTS":
+    "Tato e-mailová adresa je již používána. <a target=\"_blank\" rel=\"noopener noreferrer\" href=\"https://support.wire.com/hc/articles/115004082129\"> Další informace</a>",
```

Comment:
maybe we can increase the limit here: https://github.com/wireapp/wire-webapp/blob/dev/.prettierrc.yaml#L15

Reply:
as we did here: https://github.com/wireapp/wire-admin/blob/staging/.prettierrc.yaml#L10

Reply:
I wouldn't, as there will always be lines that break that limit.

Reply:
not if we set it to `Infinity` :P

(green to merge)